pyomo/contrib/parmest/parmest.py

```diff
@@ -68,10 +68,14 @@
 from pyomo.common.deprecation import deprecated
 from pyomo.common.deprecation import deprecation_warning
 
+from pyomo.core.base.suffix import Suffix
+
+DEPRECATION_VERSION = "6.7.2.dev0"
```

Comment:
I don't think this is actually used? But if it is being used, it's not the correct value.